internal/cli/atlas/streams/instance/delete.go

```diff
@@ -0,0 +1,93 @@
+// Copyright 2023 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package instance
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/mongodb/mongodb-atlas-cli/internal/cli"
+	"github.com/mongodb/mongodb-atlas-cli/internal/cli/require"
+	"github.com/mongodb/mongodb-atlas-cli/internal/config"
+	"github.com/mongodb/mongodb-atlas-cli/internal/flag"
+	"github.com/mongodb/mongodb-atlas-cli/internal/store"
+	"github.com/mongodb/mongodb-atlas-cli/internal/usage"
+	"github.com/spf13/cobra"
+)
+
+type DeleteOpts struct {
+	cli.GlobalOpts
+	*cli.DeleteOpts
+	store store.StreamsDeleter
+}
+
+func (opts *DeleteOpts) Run() error {
+	return opts.Delete(opts.store.DeleteStream, opts.ConfigProjectID(), opts.Entry)
+}
+
+func (opts *DeleteOpts) initStore(ctx context.Context) func() error {
+	return func() error {
+		var err error
+		opts.store, err = store.New(store.AuthenticatedPreset(config.Default()), store.WithContext(ctx))
+		return err
+	}
+}
+
+// DeleteBuilder
+// atlas streams instance delete [name].
+func DeleteBuilder() *cobra.Command {
+	opts := &DeleteOpts{
+		DeleteOpts: cli.NewDeleteOpts("Atlas Streams processor instance '%s' deleted\n", "Atlas Streams processor instance not deleted"),
+	}
+	cmd := &cobra.Command{
+		Use:   "delete <name>",
+		Short: "Delete an Atlas Stream Processor Instance",
+		Long: `The command prompts you to confirm the operation when you run the command without the --force option.
+
+An Atlas Streams processor instance with running processors cannot be deleted without stopping the processes first.
```

Comment:
reword to remove passive voice
```suggestion
Before deleting an Atlas Streams Processing instance, you must first stop all processes associated with it.
```